data/countries_names.txt

```diff
@@ -52682,49 +52682,80 @@
   uk = Ісфахан, Шираз, Ахваз
 
 [Iraq_North Description]
-  en = Mosul, Erbil, Kirkuk
-  ar = اربيل, كركوك
-  az = Mosul, Ərbil
-  da = Kirkuk
-  de = Mossul, Arbil, Kirkuk
-  el = Μοσούλη, Ερμπίλ, Κιρκούκ
-  fr = Mossoul, Erbil, Kirkouk
-  he = כירכוכ
-  ja = モースル, アルビール, キルクーク
-  pl = Mosul, Irbil, Kirkuk
-  ru = Мосул, Эрбиль, Киркук
-  sv = Kirkuk
-  tr = Musul, Erbil
-  uk = Мосул, Ербіль, Кіркук
+  en = Mosul, Erbil, Kirkuk, Tikrit
+  ar = الموصل، أربيل، كركوك، تكريت
+  az = Mosul, Ərbil, Kərkük, Tikrit
+  be = Масул, Эрбіл, Кірук, Цікрыт
+  cs = Mosul, Erbíl, Kirkúk, Tikrít
+  da = Mosul, Erbil, Kirkuk, Tikrit
+  de = Mossul, Erbil, Kirkuk, Tikrit
+  el = Μοσούλη, Ερμπίλ, Κιρκούκ, Τικρίτ
+  es = Mosul, Erbil, Kirkuk, Tikrit
+  et = Mosul, Erbil, Kirkuk, Tikrit
+  fi = Mosul, Erbil, Kirkuk, Tikrit
+  fr = Mossoul, Erbil, Kirkouk, Tikrit
+  he = מוסול, ארביל, כירכוכ, תיכרית
+  hu = Moszul, Erbíl, Kirkuk, Tikrit
+  it = Mosul, Erbil, Kirkuk, Tikrit
+  ja = モスル, エルビル, キルクーク, ティクリート
+  ko = 모술, 에르빌, 키르쿠크, 티크리트
+  nl = Mosul, Erbil, Kirkuk, Tikrit
+  pl = Mosul, Irbil, Kirkuk, Tikrit
+  pt = Mossul, Erbil, Kirkuk, Tikrit
+  ro = Mosul, Erbil, Kirkuk, Tikrit
+  ru = Мосул, Эрбиль, Киркук, Тикрит
+  sk = Mosul, Erbíl, Kirkúk, Tikrít
+  sv = Mosul, Erbil, Kirkuk, Tikrit
+  th = โมซูล, เออร์บิล, เคอร์คุก, ติกริต
+  tr = Musul, Erbil, Kerkük, Tikrit
+  uk = Мосул, Ербіль, Кіркук, Тікріт
+  vi = Mosul, Erbil, Kirkuk, Tikrit
+  bg = Мосул, Ербил, Киркук, Тикрит
+  hr = Mosul, Erbil, Kirkuk, Tikrit
+  lt = Mosulas, Erbilas, Kirkukas, Tikritas
+  lv = Mosula, Erbīla, Kirkūka, Tikrita
+  no = Mosul, Erbil, Kirkuk, Tikrit
+  sr = Мосул, Ербил, Киркук, Тикрит
+  sl = Mosul, Erbil, Kirkuk, Tikrit
+  zh = 摩苏尔, 埃尔比勒, 基尔库克, 提克里特
```

Comment:
This should be split into zh-Hans for simplified Chinese and zh-Hant for traditional Chinese here and below.

Can you please also sort lines by alphabet (en is the first)?

Reply:
My apologies for this mistake and thank you for leading me to correct my mistake. I did correct the simplified and the traditional ones, and English was already in top of the languages who follow alphabetically in order.

Reply:
No problem. Now the only change left is to sort all lines in each section by alphabet.

Reply:
> Now the only change left is to sort all lines in each section by alphabet.      
  
I sorted them by population, from the largest to the smallest. In the Iraq South section, the sorting happens to be alphabetical as well, but that was not intentional. In the Iraq North section, the cities are sorted only by size and population.

I don’t think moving Erbil and Kirkuk ahead of Mosul is appropriate, as Mosul is the second-largest city in Iraq (after Baghdad) and larger than those two.

So currently, Iraq South is sorted from the largest cities to the smallest, which also happens to be alphabetical, while Iraq North is sorted only by size and population.

Reply:
@Colonel-Dixie sorry, maybe I wasn't clear. The translations order of all other languages should be sorted:
1. en
2. af
3. ar
4. ...